docs/content/latest/yugabyte-cloud/cloud-monitor/cloud-alerts.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Cloud alerts
+linkTitle: Alerts
+description: Set alerts for activity in your cloud.
+headcontent:
+image: /images/section_icons/deploy/enterprise.png
+menu:
+  latest:
+    identifier: cloud-alerts
+    parent: cloud-monitor
+    weight: 50
+isTocNested: true
+showAsideToc: true
+---
+
+Use alerts to notify you and your team members when cluster and database resource use exceeds predefined limits, or of potential billing issues.
+
+Alerts and notifications are configured and managed on the **Alerts** page.
+
+- Yugabyte Cloud sends email notifications to all cloud account members.
+
+- Alerts are only sent once for each occurrence. For example, an alert is triggered when CPU use exceeds 75% for 10 minutes; if the condition continues for 30 minutes, you will still receive only one notification.
```

Comment:
We have following alert rules for CPU:
Warning: Node CPU Utilization exceeds 70% for 5m
Severe: Node CPU Utilization exceeds 90% for 5m